release-notes/6.0/6.0.0.md

```diff
@@ -0,0 +1,228 @@
+# Release Notes
+
+## [Stable release 6.0.0] - 2024-12-09
+
+This update brings the below changes over the previous stable release:
+
+### Breaking Changes
+
+- Removed support for .NET Standard. [#2386](https://github.com/dotnet/SqlClient/pull/2386)
+- Removed support for .NET 6 [#2927](https://github.com/dotnet/SqlClient/pull/2927)
+- Removed UWP (uap) references. [#2483](https://github.com/dotnet/SqlClient/pull/2483)
+- Removed SQL 2000 client-side debugging support for .NET Framework [#2981](https://github.com/dotnet/SqlClient/pull/2981), [#2940](https://github.com/dotnet/SqlClient/pull/2940)
+
+### Added
+
+#### JSON Support [#2916](https://github.com/dotnet/SqlClient/pull/2916), [#2892](https://github.com/dotnet/SqlClient/pull/2892), [#2891](https://github.com/dotnet/SqlClient/pull/2891), [#2880](https://github.com/dotnet/SqlClient/pull/2880), [#2882](https://github.com/dotnet/SqlClient/pull/2882), [#2829](https://github.com/dotnet/SqlClient/pull/2829), [#2830](https://github.com/dotnet/SqlClient/pull/2830)
+
+JSON Datatype support is now available in Microsoft.Data.SqlClient v6.0. This release introduces `SqlJson` type available as an extension to `System.Data.SqlDbTypes`:
+
+```csharp
+using System;
+using System.Data.SqlTypes;
+using System.Text.Json;
+
+namespace Microsoft.Data.SqlTypes
+{
+    /// <summary>
+    /// Represents the Json Data type in SQL Server.
+    /// </summary>
+    public class SqlJson : INullable
+    {
+        /// <summary>
+        /// Parameterless constructor. Initializes a new instance of the SqlJson class which 
+        /// represents a null JSON value.
+        /// </summary>
+        public SqlJson() { }
+
+        /// <summary>
+        /// Takes a <see cref="string"/> as input and initializes a new instance of the SqlJson class.
+        /// </summary>
+        /// <param name="jsonString"></param>
+        public SqlJson(string jsonString) { }
+
+        /// <summary>
+        /// Takes a <see cref="JsonDocument"/> as input and initializes a new instance of the SqlJson class.
+        /// </summary>
+        /// <param name="jsonDoc"></param>
+        public SqlJson(JsonDocument jsonDoc) { }
+
+        /// <inheritdoc/>
+        public bool IsNull => throw null;
+
+        /// <summary>
+        /// Represents a null instance of the <see cref="SqlJson"/> type.
+        /// </summary>
+        public static SqlJson Null { get { throw null; } }
+
+        /// <summary>
+        /// Gets the string representation of the Json content of this <see cref="SqlJson" /> instance.
+        /// </summary>
+        public string Value { get ; }
+    }
+}
+```
+
+JSON datatype is supported by the driver for reading, writing, streaming and performing bulk copy operations.
+
+#### Introducing SqlClientDiagnostics [#2226](https://github.com/dotnet/SqlClient/pull/2226)
```

Comment:
Please update this based on latest changes from https://github.com/MicrosoftDocs/sql-docs-pr/pull/32771